src/editors/containers/ProblemEditor/components/EditProblemView/index.jsx

```diff
@@ -128,6 +129,7 @@ const EditProblemView = ({ returnFunction }) => {
             </Container>
           ) : (
             <span className="flex-grow-1 mb-5">
+              <ProblemEditorPluginSlot blockType={problemType || 'problem'} />
```

Comment:
Could we use something like `blockTypes.problem` instead of a hardcoded value?

```suggestion
              <ProblemEditorPluginSlot blockType={problemType || blockTypes.problem} />
```